vignettes/articles/analysis.qmd

```diff
@@ -0,0 +1,627 @@
+
+## ANOVA-Type Models
+
+The analysis of variance (ANOVA) is a bedrock tool for comparing different groups in a data set. Our outcome scores are categories, so a type of model called a *generalized linear model* can be used to analyze the results. Without ordered categories, we'll focus on an ordinal logistic regression model. There are many different variants of this model but one of the most common is the cumulative probability model. For $k=1\ldots c$ possible values of the scores, there are $c-1$ probabilities $Pr[Y_{i} \ge k]$ for $i=1\ldots n$ samples. For example, $Pr[Y_{i} \ge 2]$ would model the probability of being not completely correct.
+
+<!--# TODO: I _think_ that last probability is actually the probably of being not completely incorrect, i.e. the probability of 2 (Patially Correct) or 3 (Correct) -->
+
+### Independent Data
+
+Let's first consider the case where each of the rows in `are_eval` are statistically independent, conditional on `model`. Said another way, we can keep only the rows for the first epoch:
+
+```{r}
+#| label: single-epoch-data
+are_eval_single <- are_eval %>% filter(epoch == 1)
+
+are_eval_single
+```
+
+In this setting, we could visualize scores sample-by-sample for one epoch like so:
+
+```{r}
+#| label: single_sample
+are_eval_single %>%
+  ggplot(aes(x = model, y = id)) +
+  geom_raster(aes(fill = score), alpha = 2 / 3) +
+  scale_fill_brewer(palette = "Set1") +
+  labs(x = NULL, y = NULL) +
+  theme(legend.position = "top")
+```
+
+To model the score probabilities, the cumulative logit model estimates $c-1$ values of
+
+$$
+\log\left(\frac{Pr[Y_{i} \ge k]}{1 - Pr[Y_{i} \ge k]]}\right) = \theta_k - (\beta_2x_{i2} + \ldots +  \beta_{p}x_{ip})
+$$
+
+for $i=1\ldots n$ samples evaluated with $j=2\ldots p$ LLMs which produce outcomes $k=1\ldots c-1$. The $x_{ij}$ values are binary indicators for all but one of the LLMs. This is a full-rank parameterization; the effect of the LLM corresponding to $j=1$ is captured by the different intercepts ($\theta_k$). With this parameterization, the $\beta_j$ parameters correspond to the effect of the LLM $j$ above and beyond the effect of the LLM associated with $j=1$.
+
+This is a *parallel* model because it assumes that the pattern of LLM effectiveness is constant across the different outcome categories; in other words, if Model A is twice as likely as Model B to get a "Correct" vs "Partially correct" answer, then Model A is also twice as likely to get "Partially correct or Correct" vs "Incorrect". It can also referred to as a proportional odds model.
+
+<!--# TODO: is this new "parallel" interpretation correct? -->
```

Comment:
Was a bit unclear on the "parallel" bit so added some additional explanation. Is this fine to resolve?